app/assets/onepager/css/main.css

```diff
@@ -560,6 +560,18 @@ body #main a:not(.button):hover {
   margin: auto;
 }
 
+.alert {
+  position: absolute;
+}
+
+#footer {
+  position: absolute;
+  background-color: #0D0764;
```

Comment:
for real center the content of this we can do 
left: 0;  right: 0;  text-align: center
In that way not matter the strength of the content will always be centered

Reply:
I keep seeing 100vw here not sure what do you want to solve here @iamonuwa 
What is already on production is perfectly centered and why putting the bg blue of this footer if is actually transparent?
![image](https://user-images.githubusercontent.com/1358093/65105621-e2d8ba00-d9ab-11e9-9b9d-a1c8968e4917.png)